src/input.rs

```diff
@@ -108,6 +112,41 @@ pub fn touches_local() -> Vec<Touch> {
         .collect()
 }
 
+fn update_mouse_touch_if_necessary() {
```

Comment:
I feel like this doesn't belong to the documentation PR?

Reply:
Yes, that is not documentation, but rather a new feature, should I move it into another branch?

Reply:
yes please

Reply:
Ok, I will try to do that later